spec/unit_tests/moab/storage_object_version_spec.rb

```diff
@@ -1,58 +1,71 @@
 # frozen_string_literal: true
 
 describe Moab::StorageObjectVersion do
-  before(:all) do
-    @temp_dir_ingests = temp_dir.join("ingests")
-    @temp_object_dir = @temp_dir_ingests.join(BARE_TEST_DRUID)
-    @existing_object_pathname = ingests_dir.join(BARE_TEST_DRUID)
-    @existing_storage_object = Moab::StorageObject.new(FULL_TEST_DRUID, @existing_object_pathname)
-    @existing_storage_object_version = described_class.new(@existing_storage_object, 2)
-    @temp_storage_object = Moab::StorageObject.new(FULL_TEST_DRUID, @temp_object_dir)
-    @temp_package_pathname = temp_dir.join("packages")
+  let(:temp_dir_ingests) { temp_dir.join("ingests") }
+  let(:temp_object_dir) { temp_dir_ingests.join(BARE_TEST_DRUID) }
+  let(:existing_object_pathname) { ingests_dir.join(BARE_TEST_DRUID) }
+  let(:existing_storage_object_version) do
+    existing_storage_object = Moab::StorageObject.new(FULL_TEST_DRUID, existing_object_pathname)
+    described_class.new(existing_storage_object, 2)
+  end
+  let(:temp_storage_object) { Moab::StorageObject.new(FULL_TEST_DRUID, temp_object_dir) }
+  let(:version_with_manifest_errors) do
     bad_object_pathname = temp_dir.join(BARE_TEST_DRUID)
     bad_object_pathname.rmtree if bad_object_pathname.exist?
     bad_object_pathname.mkpath
-    FileUtils.cp_r(@existing_object_pathname.join('v0001').to_s, bad_object_pathname.join('v0001').to_s)
-    @object_with_manifest_errors = Moab::StorageObject.new(FULL_TEST_DRUID, bad_object_pathname)
-    @version_with_manifest_errors = @object_with_manifest_errors.storage_object_version(1)
-    new_manifest_file = @version_with_manifest_errors.version_pathname.join('manifests', 'dummy1.xml')
+    FileUtils.cp_r(existing_object_pathname.join('v0001').to_s, bad_object_pathname.join('v0001').to_s)
```

Comment:
I don't know how far you're going with the refactor, but I'd expect to see file copies in a `before` not a `let`.

Reply:
it's because it's creating "version_with_manifest_errors" by doing the copy ...